behat/features/threshold.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Check threshold functionally
+  In order to be able to create CI build items with error state
+  As a privileges user
+  I need to see new status of CI build and new incidents.
+
+  @api
+  Scenario: Check create new repository.
+    Given I login with user "William"
+    When  I create repository "William/app"
+    And   I visit repository "William/app"
```

Comment:
`Then I should have access to repository "William/app"`

(i.e. you can skip the I visit, and use `node_access()`)